src/components/pools/common/InvestorsModal.tsx

```diff
@@ -145,22 +145,35 @@ const InvestorsTable = genericSuspense(
   ({ pool }: { pool: ParsedAelinPool }) => {
     const router = useRouter()
     const [orderDirection, setOrderDirection] = useState<OrderDirection>(OrderDirection.Desc)
+    const [sortBy, setSortBy] = useState<Investor_OrderBy | undefined>(
+      Investor_OrderBy.AmountInvested,
+    )
 
     const { data, hasMore, nextPage } = useAelinInvestors({
-      orderDirection: OrderDirection.Desc,
+      orderBy: sortBy,
+      orderDirection: orderDirection,
       where: {
         poolAddress: pool.address,
       },
     })
 
-    const handleSort = () => {
-      if (orderDirection === OrderDirection.Desc) return setOrderDirection(OrderDirection.Asc)
-      setOrderDirection(OrderDirection.Desc)
+    const handleSort = (orderBy: Investor_OrderBy | undefined) => {
```

Comment:
Maybe `newSortBy` instead of `orderBy` will be more clear?

Reply:
Makes sense. It's done here d57094d6663c1b4e5c15e1b1eff76f33284012d1